src/backend/metrics/metrics.ts

```diff
@@ -246,11 +252,12 @@ export async function trackPidPlaytime(
       trackEvent({
         event: 'Game Launched',
         properties: {
-          isBrowserGame: false,
+          isBrowserGame: gameInfo.browserUrl !== undefined,
```

Comment:
Maybe we could just have it `Boolean(gameInfo.browserUrl)` at least, since we need to consider if `browserUrl` could be empty str or such

Reply:
I am actually not sure if we need that since we are passing the platform now. But I need to be sure that this is not being used by any dashboard on mixpanel.

Reply:
I guess we can just remove to be honest, because since we know the platform and we know the game to what belongs if is browse or not 🙌

Reply:
Actually, this method is used only for native games so doesn't need any change. I reverted these.